models.py

```diff
@@ -29,7 +29,7 @@ class User(Base):
     __tablename__ = 'users'
     id = Column(Integer, primary_key=True)
     name = Column(String(50), unique=True)
-    email = Column(String(120), unique=True)
```

Comment:
Email addresses are usually way longer than 5 characters, so we should actually change that back.